addons/tmf_loadouts/loadouts/us_rangers_2010.hpp

```diff
@@ -277,8 +278,7 @@ class mmgtl : aar {
 		"ItemMap",
 		"ItemCompass",
 		"ItemWatch",
-		"ACE_Vector",
-		"ItemGPS"
+		"ACE_Vector"

```

Comment:
Adding a vector to the gunner?

Reply:
Nope, vector existed in the last iteration. Just removing the GPS and replacing it with a microDagr in order to provide increased functionality (Vector can link with microDagr and can pre-plot distance between points on the microDagr, etc. etc.).